src/main/java/com/porpapps/wealthwatchapi/model/Portfolio.java

```diff
@@ -0,0 +1,13 @@
+package com.porpapps.wealthwatchapi.model;
+
+import java.time.LocalDate;
+
+public class Portfolio {
```

Comment:
Let's also add getters, setters, a no-args constructor, toString, equals, and hash code methods. For that you can just add an annotation `@Data` in the classes you created

Reply:
done